src/main/java/net/greghaines/jesque/worker/Watchdog.java

```diff
@@ -41,28 +48,39 @@
             Executors.newScheduledThreadPool(POOL_SIZE);
     private final AtomicReference<String> watchdogScriptHash = new AtomicReference<>(null);
     private Pool<Jedis> jedisPool;
+    private boolean redisRestartRecoveryOn;
+    private Runnable listener;
+    private String serverName;
+    private String isDebug;
```

Comment:
Can we save this one?
If not - can we make it boolean?

Reply:
We  can , but then we will need to convert it to string every time we call lua script. I went for performance , not for readability , but it can be changed easily ...